tests/typing_test_data.py

```diff
@@ -35,14 +35,18 @@ def f5(x: int, *, y: str, z: str) -> None:
     pass
 
 
-def f6(x: int = None, y: dict = {}) -> None:
```

Comment:
Keeping them properly grouped. I can append instead of inserting if that makes you happy.